mtime.go

```diff
@@ -1,14 +1,16 @@
 // Package mtime provides time operations using a monotonic time source, which
 // is useful when you want to work with elapsed rather than wall time. Based on
-// github.com/aristanetworks/monotime.
+// github.com/aristanetworks/monotime. mtime uses Instants. INSTANTS ARE NOT
+// REAL TIMES, they are only useful for measuring elapsed time.
 package mtime
 
 import (
 	"github.com/aristanetworks/goarista/monotime"
 	"time"
 )
 
-// An Instant represents an instant in monotonically increasing time
+// An Instant represents an instant in monotonically increasing time. INSTANTS
+// ARE NOT REAL TIMES, they are only useful for measuring elapsed time.
 type Instant uint64
```

Comment:
@joesis We still need Instant because we actually store the value atomically in some places for measurement purposes. I just removed the Time() method and added a warning to prevent people from abusing Instant.